code/drasil-example/projectile/lib/Drasil/Projectile/Concepts.hs

```diff
@@ -31,15 +31,16 @@ projMotion = compoundNC projectile motion
 ---
 
 defs :: [ConceptChunk]
-defs = [launcher, projectile, target, projSpeed]
+defs = [launcher, projectile, target, projSpeed, projPos]
 
-launcher, projectile, target, projSpeed :: ConceptChunk
+launcher, projectile, target, projSpeed, projPos :: ConceptChunk
 launcher   = dcc "launcher"   (nounPhraseSP "launcher")  ("where the projectile is launched from " ++
                                                           "and the device that does the launching")
 projectile = dcc "projectile" (nounPhraseSP "projectile") "the object to be launched at the target"
 target     = dcc "target"     (nounPhraseSP "target")     "where the projectile should be launched to"
 
-projSpeed  = dccWDS "projSpeed" (nounPhraseSP "1D speed") (S "1D speed under" +:+ phrase constant +:+ phrase acceleration)
+projSpeed  = dccWDS "projSpeed" (nounPhraseSP "1D speed")    (getAcc oneD +:+ S"speed under" +:+ phrase constant +:+ phrase acceleration)
+projPos    = dccWDS "projPos"   (nounPhraseSP "1D position") (getAcc oneD +:+ S"position under" +:+ phrase constant +:+ phrase speed)
```

Comment:
A minor nitpick is that there should be a space after the `S` constructor. Other than that, this looks good!

Reply:
Also, I just realized the concepts for `speed` and `position` should be used as well